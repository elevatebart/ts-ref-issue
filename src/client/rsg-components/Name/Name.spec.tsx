import React from 'react';
import { shallow } from 'enzyme';
import { NameRenderer, styles } from './NameRenderer';

const props = {
	classes: classes(styles),
};

it('renderer should render argument name', () => {
	const actual = shallow(<NameRenderer {...props}>Foo</NameRenderer>);

	expect(actual).toMatchSnapshot();
});

it('renderer should render deprecated argument name', () => {
	const actual = shallow(
		<NameRenderer {...props} deprecated>
			Foo
		</NameRenderer>
	);

	expect(actual).toMatchSnapshot();
});
