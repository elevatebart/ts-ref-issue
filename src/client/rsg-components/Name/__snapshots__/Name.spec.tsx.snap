// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer should render argument name 1`] = `
<code
  className="name"
>
  Foo
</code>
`;

exports[`renderer should render deprecated argument name 1`] = `
<code
  className="name isDeprecated"
>
  Foo
</code>
`;
